core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -415,9 +416,9 @@ public void testAddSnapshotFromJson() throws IOException {
     assertEquals(action, expected, MetadataUpdateParser.fromJson(json));
   }
 
-  /** RemoveSnapshots * */
+  /** RemoveSnapshot * */
   @Test
-  public void testRemoveSnapshotsFromJson() {
+  public void testRemoveSnapshotFromJson() {
```

Comment:
Nit: Normally I'd say we shouldn't really make this change to the different test names and just prefer to keep it as is. I understand the intent so it's OK but the assertions in this test will anyways change in the future when RemoveSnapshot is removed. It'd make more sense to change it at that point.